backend/src/main/java/com/github/gadenko/gamereviewsite/backend/service/GameReviewService.java

```diff
@@ -22,31 +23,39 @@ public List<GameReview> getAllGameReviews() {
 
     public GameReview getGameReviewById(String id) {
         return gameReviewRepo.findById(id)
-                .orElseThrow( () -> new NoSuchElementException("Gamereview not found with id: " + id));
+                .orElseThrow(() -> new NoSuchElementException("Gamereview not found with id: " + id));
     }
 
     public GameReview addNewGameReview(CreateGameReviewDto newGameReview) {
         GameReview newReview = new GameReview();
-        if(newGameReview.getTitle() == null){
+        if (newGameReview.getTitle() == null) {
             throw new IllegalArgumentException("Title of the given gamereview was null");
         }
-        if(newGameReview.getHeadline() == null){
+        if (newGameReview.getHeadline() == null) {
             throw new IllegalArgumentException("Headline of the given gamereview was null");
         }
-        if(newGameReview.getGameDescription() == null){
+        if (newGameReview.getGameDescription() == null) {
             throw new IllegalArgumentException("GameDescription of the given gamereview was null");
         }
-        if(newGameReview.getPicture() == null){
+        if (newGameReview.getPicture() == null) {
             throw new IllegalArgumentException("Picture of the given gamereview was null");
         }
-        if(newGameReview.getCategory() == null){
+        if (newGameReview.getCategory() == null) {
             throw new IllegalArgumentException("Category of the given gamereview was null");
         }
+        if (newGameReview.getGraphic() == null) {
+            throw new IllegalArgumentException("Graphic of the given gamereview was null");
+        }
+        if (newGameReview.getSound() == null) {
+            throw new IllegalArgumentException("Sound of the given gamereview was null");
+        }
         newReview.setTitle(newGameReview.getTitle());
         newReview.setHeadline(newGameReview.getHeadline());
         newReview.setGameDescription(newGameReview.getGameDescription());
         newReview.setPicture(newGameReview.getPicture());
         newReview.setCategory(newGameReview.getCategory());
+        newReview.setGraphic(newGameReview.getGraphic());
+        newReview.setSound(newGameReview.getSound());
         return gameReviewRepo.insert(newReview);
     }
```

Comment:
Der ganze Block wirkt recht komplex. Hast du mal versucht, das über die verkürzte Schreibweise zu machen.
Beispiel:

newReview.setTitle(newGameReview.getTitle() ? newReview.setTitle(newGameReview.getTitle() : throw new IllegalArgumentException("Title of the given gamereview was null"));